vite.config.ts

```diff
@@ -0,0 +1,7 @@
+import { defineConfig } from 'vite'
+import react from '@vitejs/plugin-react'
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [react()],
```

Comment:
A useful plugin for Typescript types' checking: [vite-plugin-checker](https://vite-plugin-checker.netlify.app/)